atom/common/native_mate_converters/net_converter.cc

```diff
@@ -68,11 +69,72 @@ v8::Local<v8::Value> Converter<scoped_refptr<net::X509Certificate>>::ToV8(
             val->GetIntermediateCertificates().front(),
             issuer_intermediates);
     dict.Set("issuerCert", issuer_cert);
+
+    std::vector<std::string> intermediates_encoded;
+    for (size_t i = 0; i < intermediates.size(); i++) {
+      auto native_cert = intermediates[i];
+      std::string encoded;
+      net::X509Certificate::GetPEMEncoded(native_cert, &encoded);
```

Comment:
Would it make sense for `intermediates` to be an array of objects though that is the full cert information instead of just the encoded part?

I think it could just reuse what `dict.Set("issuerCert", issuer_cert);` does.